### PR TITLE
Get domain and token according to test environment

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

**Review by ggallien13 — APPROVED** [review]

File: packages/pangea-sdk/pangea/tools_util.py [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
-from datetime import datetime, timezone, date
+from datetime import date, datetime, timezone
 
 from pangea.config import PangeaConfig
 from pangea.services import Audit
 
 
-class Root(t.TypedDict):
+class TestEnvironment(str, enum.Enum):
+    PRODUCTION = "PROD"
```
Comment (doubletooth) [inline packages/pangea-sdk/pangea/tools_util.py:17]: we use dev/stg/lve in the pangea repo, might be worth using the same nomenclature here?

File: packages/pangea-sdk/pangea/tools_util.py [hunk 2]
```diff
@@ -1,26 +1,39 @@
...
+    DEVELOP = "DEV"
```
Comment (doubletooth) [inline packages/pangea-sdk/pangea/tools_util.py:18]: nit: would prefer these be alphabetized

File: packages/pangea-sdk/pangea/tools_util.py [hunk 3]
```diff
@@ -104,12 +117,16 @@ def json_defaults(obj):
...
-        k: v if not isinstance(v, dict) else filter_deep_none(v)
-        for k, v in data.items()
-        if v is not None
-    }
+def filter_deep_none(data: t.Dict) -> t.Dict:
+    return {k: v if not isinstance(v, t.Dict) else filter_deep_none(v) for k, v in data.items() if v is not None}
+
+
+def get_test_domain(environment: TestEnvironment):
+    return os.getenv(f"PANGEA_INTEGRATION_DOMAIN_{environment}")
```
Comment (doubletooth) [inline packages/pangea-sdk/pangea/tools_util.py:125]: nit: I prefer just failing loudly in cases where environment variables don't exist and they're needed. makes debugging easier

File: packages/pangea-sdk/pangea/tools_util.py (hunk 1 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/pangea/tools_util.py:17]: @ggallien13 thoughts? I don't have any problem to update all SDKs to that nomenclature, but I should need you to update env var

File: packages/pangea-sdk/pangea/tools_util.py (hunk 3 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/pangea/tools_util.py:125]: Agree. I'll add that

File: packages/pangea-sdk/pangea/tools_util.py (hunk 1 above)
Comment (ggallien13) [reply packages/pangea-sdk/pangea/tools_util.py]: Eh, I prefer PROD, but it's good to be consistent. Let's change it to LVE.

**Review by ggallien13 — APPROVED** [review]

**Pull request merged**